src/test/java/tests/NavigationMenuTest.java

```diff
@@ -0,0 +1,30 @@
+package tests;
+
+import constants.NavigationLinksConstants;
+import constants.NewsMenuItemsConstants;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class NavigationMenuTest extends BaseTest {
+
+    @DataProvider(name = "menuItemName")
+    public Object[][] menuItemName() {
+        return new Object[][]{{NewsMenuItemsConstants.PEOPLE}, {NewsMenuItemsConstants.CARS}, {NewsMenuItemsConstants.TECHNOLOGIES}, {NewsMenuItemsConstants.REAL_ESTATE}};
+    }
+
+    @Test(dataProvider = "menuItemName")
+    public void openNavigationLink_menuItemsShouldBePresent(String menuItemName) {
+        navigationStep.moveToNavigationLinkTooltip(NavigationLinksConstants.NEWS)
```

Comment:
This test contains just one action... what do we validate here? It's more like just a test step not a test itself